packages/dev-server-swc/index.mjs

```diff
@@ -0,0 +1,6 @@
+// this file is autogenerated with the generate-mjs-dts-entrypoints script
```

Comment:
if this is autogenerated, it should be gitignored

Reply:
I copied this over from the `dev-server-esbuild` plugin which doesn't seemed to be gitignored either. In fact all the `index.mjs` files are not gitignored.